lib/widgets/drop_down.dart

```diff
@@ -1,42 +1,52 @@
 import 'package:dropdown_button2/dropdown_button2.dart';
 import 'package:flutter/material.dart';
 
-class DropDown extends StatefulWidget {
-  List<String> items;
+class DropDown extends StatelessWidget {
+  final String? dropdownValue;
+  final List<String> items;
+  final Function didSelectItemHandler;
 
-  DropDown(this.items, {super.key});
+  const DropDown(
+      {required this.items,
+      required this.didSelectItemHandler,
+      this.dropdownValue,
+      super.key});
 
   @override
-  State<DropDown> createState() => _DropDownState();
-}
-
-class _DropDownState extends State<DropDown> {
-  String dropdownValue = "swift";
-  @override
-  Widget build(BuildContext context) => DropdownButton2(
-        isExpanded: true,
-        hint: Text(
-          'Select Item',
-          style: TextStyle(
-            fontSize: 14,
-            color: Theme.of(context).hintColor,
+  Widget build(BuildContext context) => Container(
```

Comment:
Could you check if you can replace Container here as well?

Reply:
removed container